src/pages/workspace/categories/WorkspaceCategoriesPage.tsx

```diff
@@ -309,22 +309,22 @@ function WorkspaceCategoriesPage({route}: WorkspaceCategoriesPageProps) {
                     danger
                 />
                 {shouldUseNarrowLayout && <View style={[styles.pl5, styles.pr5]}>{getHeaderButtons()}</View>}
-                {(!shouldUseNarrowLayout || shouldShowEmptyState || isLoading) && getHeaderText()}
+                {(!shouldUseNarrowLayout || (!hasVisibleCategories && !isLoading) || isLoading) && getHeaderText()}
```

Comment:
`&& !isLoading` condition is redundant.
Also, can we make 2 pages consistent?
One is `!hasVisibleCategories` , another is `tagList.length === 0`